IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -142,16 +142,6 @@ - (void)cancel
     NSError *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorSessionCanceledProgrammatically, @"Authorization session was cancelled programatically.", nil, nil, nil, self.context.correlationId, nil);
     
     [_telemetryEvent setIsCancelled:YES];
-    
-    if ([self.webView.navigationDelegate isEqual:self])
-    {
-        /*
-         There's an edge case scenario, where application cancels one of the requests and starts another one immediately after.
-         We want to make sure that second request doesn't get first response, so we cancel loading of webview when cancel method is called         
-         */
-        [self.webView stopLoading];
```

Comment:
I used to test the scenario of double-clicking the "register" page, where I have the same conclusion. `stopLoading` could not stop the previous response coming to the webview, unless we use a new webview every time. Maybe we could consult with Apple to see what is the recommended way to do it if `stopLoading` doesn't do the trick :)